test/lender/Lender.sol

```diff
@@ -174,11 +174,12 @@ contract Lender {
   /// @param u address of an underlying asset
   /// @param m maturity (timestamp) of the market
   /// @param a the amount of underlying tokens to lend
-  /// @param mb the minimum amount of zero-coupon tokens to return accounting for slippage
+  /// @param minimumBought the minimum amount of zero-coupon tokens to return accounting for slippage
   /// @param d the maximum timestamp at which the transaction can be executed
-  function lend(uint8 p, address u, uint256 m, uint256 a, uint256 mb, uint256 d) public returns (uint256) {
+  function lend(uint8 p, address u, uint256 m, uint256 a, uint256 minimumBought, uint256 d) public returns (uint256) {
```

Comment:
can `minimumBought` just be `r` ?

don't confuse our bitching about Element's `return_` var with this.

it **IS** the min amount of ZCT to return right? i mean the comment says so.. i like `r`

Reply:
Yes it is, this can be `r`.